audit-logs/chart/templates/_ceph-config.tpl

```diff
@@ -0,0 +1,94 @@
+{{- define "ceph.labels" }}
```

Comment:
As suggested, first focus on the audit logs within Kubernetes to enable SIEM/SEM.